datahub-classify/src/datahub_classify/infotype_helper.py

```diff
@@ -400,7 +412,10 @@ def inspect_for_full_name(metadata, values, config):  # noqa: C901
                 weight = 1
                 for value in values:
                     try:
-                        if len(value) <= 50:
+                        if (
+                            len(value) <= 50
+                            and len(re.split(r"[^a-zA-Z0-9]", value)) >= 2
```

Comment:
Can use the regex whitespace operator instead? `\s+`

Reply:
Here we are considering all characters except alpha-numeric

Reply:
This will have issues if the name ever contains special characters like `é`

It would be better to modify the regex to split on whitespace and other punctuation

Reply:
Modified the regex to split on \s and other punctuation